Sources/YapDB.swift

```diff
@@ -443,19 +449,19 @@ extension YapDB {
         func setup() -> YapDatabaseSecondaryIndexSetup {
             let setup = YapDatabaseSecondaryIndexSetup()
             for (column, indexType) in columnTypes {
-                setup.addColumn(column, withType: indexType)
+                setup.addColumn(column, with: indexType)
             }
             return setup
         }
 
-        public func registerInDatabase(database: YapDatabase, withConnection connection: YapDatabaseConnection?) {
+        open func registerInDatabase(_ database: YapDatabase, withConnection connection: YapDatabaseConnection?) {
```

Comment:
I'm unsure why someone would want to override this, but I don't see the harm in it and I'd prefer to err on the side of extensibility when unsure.